framework/src/main/java/org/checkerframework/common/basetype/messages.properties

```diff
@@ -2,6 +2,7 @@
 receiver.invalid=incompatible types.%nfound   : %s%nrequired: %s
 type.anno.before.modifier=write type annotation %s immediately before type, after modifiers %s
 type.anno.before.decl.anno=write type annotations %s immediately before type, after declaration annotation %s
+new.class.type.invalid=the annotations %s do not need be applied in object creations
```

Comment:
Can you move this further down to other `type.invalid` error messages.
While we're at it, can you improve the grammar?